content/en/Getting started/details.md

```diff
@@ -10,8 +10,11 @@ description: >
 Some detail requirements vary by the type of asset.
 {{% /pageinfo %}}
 
-Our pentests have common requirements for all assets, as well as requirements for
-specific assets. 
+<!-- For content, see https://github.com/cobalthq/cobalt-product-public-docs/blob/main/layouts/shortcodes/big-pic-UI-steps.md -->
```

Comment:
Until the PR is merged, you'll see this file in the following directory: https://github.com/cobalthq/cobalt-product-public-docs/tree/DOCS-70_incorporate_5steps_article_from_zendesk/layouts/shortcodes